app/controllers/application_controller.rb

```diff
@@ -10,6 +10,8 @@ class ApplicationController < ActionController::Base
   helper_method :current_user_or_visitor
   helper_method :dashboard_or_root_path
   helper_method :subdomain
+  helper_method :show_max_size_warning?
+  helper_method :show_max_size_reached?
```

Comment:
Does this need to be all the way up in ApplicationController? What about BaseController? Or separating out into a GroupInvitations concern or something? No need to pollute this namespace, I don't think.